LayoutTests/editing/spelling/inline-spelling-markers-hidpi.html

```diff
@@ -21,6 +21,7 @@
 </head>
 
 <body>
+<script src="../../resources/ui-helper.js"></script>
 <script src="../../resources/js-test-pre.js"></script>
```

Comment:
Nit: While here, consider updating these tests to use `js-test.js` rather `-pre.js` and `-post.js`.

Reply:
Will do — thanks for the review!